ktor-server/ktor-server-core/mingwX64/src/io/ktor/server/engine/EnvironmentUtils.mingwX64.kt

```diff
@@ -16,3 +16,23 @@ internal actual fun setEnvironmentProperty(key: String, value: String): Unit = m
 internal actual fun clearEnvironmentProperty(key: String): Unit = memScoped {
     SetEnvironmentVariable!!(key.wcstr.ptr, null)
 }
+
+@OptIn(ExperimentalForeignApi::class)
+internal actual fun getKtorEnvironmentProperties(): List<Pair<String, String>> = buildList {
+    val rawEnv: LPWCH = GetEnvironmentStringsW() ?: return@buildList
+    try {
+        var current: CPointer<WCHARVar>? = rawEnv
+        while (current != null) {
+            val keyValue = current.toKString()
+            if (keyValue.isEmpty()) break
+            current += keyValue.length + 1
+
+            val (key, value) = keyValue.splitPair('=') ?: continue
+            if (key.startsWith("ktor.")) {
+                add(key to value)
+            }
```

Comment:
Just curious. Is it intentionally slightly different from the code in nix source-set?

```kotlin
val (key, value) = keyValue.splitPair('=') ?: continue
if (key.startsWith("ktor.")) {
    add(key to value)
}
```
vs
```kotlin
if (keyValue.startsWith("ktor.")) {
    val (key, value) = keyValue.splitPair('=') ?: continue
    add(key to value)
}
```

Reply:
nope, didn't noticed it